### PR TITLE
Update prismic module

### PR DESCRIPTION
**Pull request by paul-vd**
closes FC-837

Preview: https://deploy-preview-463--heuristic-almeida-1a1f35.netlify.app/docs/appendices/migration-guides/#the-prismic-module-has-moved-to-a-new-home

### CODE REVIEW

**Review by real34 — APPROVED** [review]

File: docs/appendices/migration-guides/index.mdx [hunk 1]
```diff
@@ -14,6 +14,69 @@ import TabItem from "@theme/TabItem";
...
+
+We have tagged the last version of the Prismic module in the
+[external repository](https://gitlab.com/front-commerce/front-commerce-prismic)
+`v1` 🥳
+
+Since this version we have moved the Prismic module
+[into the core repository](https://gitlab.com/front-commerce/front-commerce/-/tree/main/modules/prismic)
+of Front-Commerce, moving forward any new features will be made available and
+follow the same
+[semantic versioning](http://localhost:3002/docs/appendices/release-process#semantic-versioning)
```
Comment (Kagaminara) [inline docs/appendices/migration-guides/index.mdx:27]: ⚠️  The link points to localhost

**Review by Kagaminara — CHANGES_REQUESTED** [review]

File: docs/appendices/migration-guides/index.mdx [hunk 2]
```diff
@@ -14,6 +14,69 @@ import TabItem from "@theme/TabItem";
...
+
+You can use the module by adding the following changes to your
+`.front-commerce.js` config file:
+
+```diff title=".front-commerce.js"
+module.exports = {
+  modules: [
+    "./node_modules/front-commerce/theme-chocolatine",
+-   "./node_modules/front-commerce/modules/prismic",
++   "./node_modules/front-commerce-prismic/prismic",
```
Comment (Kagaminara) [inline docs/appendices/migration-guides/index.mdx:38]: Shouldn't it be like this? I thought the front-commerce-prismic was now deprecated 🤔 If not I think there is some confusion in the description above.
```suggestion
-   "./node_modules/front-commerce-prismic/prismic",
+   "./node_modules/front-commerce/modules/prismic",
```

File: docs/appendices/migration-guides/index.mdx (hunk 1 above)
Comment (real34) [reply docs/appendices/migration-guides/index.mdx:27]:
```suggestion
[semantic versioning](/docs/appendices/release-process#semantic-versioning)
```

**Review by real34 — CHANGES_REQUESTED** [review]

File: docs/appendices/migration-guides/index.mdx (hunk 2 above)
Comment (paul-vd) [reply docs/appendices/migration-guides/index.mdx:38]: I switched them when creating the diff 🤦 thanks

**Pull request merged**